pages/index.js

```diff
@@ -45,8 +45,16 @@ export default class extends React.Component {
   render() {
     const { posts, featuredPost, postCount } = this.props;
 
+    var iframeStyle = {
+      display: 'none',
+      visibility: 'hidden'
+    };
+    
     return (
       <Fragment>
+        <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TZK32GR"
```

Comment:
This will only show up on `/`, if this is not intentional, include it at the end of `pages/_app.js`.

Also, please clean it up a bit please:) (you can also define the style inline if you want to)
```html
<noscript>
  <iframe
    src="https://www.googletagmanager.com/ns.html?id=GTM-TZK32GR"
    height="0"
    width="0"
    style={{ display: 'none', visibility: 'hidden' }}
  />
</noscript>
```